index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./index.js"></script>
+    <link rel="stylesheet" href="./master.css">
+    <title>Studio Gibby</title>
+</head>
+<body>
+    <header>
+        <h1>Ghibli Review App</h1>
+             <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" alt="">
+     </header>
+    
+<select name="" id=""></select>
```

Comment:
You need to add the first default blank selection (or saying "Pick a film" might be more user friendly) by writing 
`<select><option disabled selected>Pick a film</option></select>`